lib/internal/Magento/Framework/Module/ModuleList/Loader.php

```diff
@@ -85,6 +107,26 @@ public function load()
     }
 
     /**
+     * Returns a traversable yielding content of all module.xml files
+     *
+     * @return \Traversable
+     *
+     * @author Josh Di Fabio <joshdifabio@gmail.com>
+     */
+    private function getModuleConfigs()
+    {
+        $modulesDir = $this->filesystem->getDirectoryRead(DirectoryList::MODULES);
+        foreach ($modulesDir->search('*/*/etc/module.xml') as $filePath) {
+            yield $modulesDir->readFile($filePath);
```

Comment:
Wow, that's the first usage of [generators](http://us3.php.net/manual/en/language.generators.php) in Magento 2 :)